pkg/apis/forklift/v1beta1/register.go

```diff
@@ -33,3 +36,19 @@ var SchemeGroupVersion = schema.GroupVersion{
 }
 
 var SchemeBuilder = &scheme.Builder{GroupVersion: SchemeGroupVersion}
+
+// TODO: find a better place for this, it would be nice to use it also to
+// determine the plural configuration of a resource for the operator
+func GetGroupResource(required runtime.Object) (group string, resource string, err error) {
+
+	switch required.(type) {
+	case *Provider:
+		group = SchemeGroupVersion.Group
+		resource = "providers"
+	default:
```

Comment:
shouldn't we add more types? user should query only providers? what about other resources?

Reply:
there's no need to add more at the moment - the inventory handles only 'providers' (and from those it takes data such as mappings), other endpoints are handled by the 'infrastructure'